src/InvocationRegistry.sol

```diff
@@ -370,6 +370,11 @@ contract InvocationRegistry is OwnableUpgradeable, UUPSUpgradeable {
     }
 
     function _invoke(uint256 orbId, address invoker_, bytes32 contentHash_) internal virtual {
+        (, address purchaser,,) = os.ownership().purchaseOrder(orbId);
+        if (purchaser != address(0)) {
+            revert NotInvokable();
+        }
```

Comment:
Would maybe add a small comment here as a reminder for why this rule exists 👍

i.e. `keeper shouldn't be able to invoke the orb if there's a standing order to buy it`